openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -29,6 +38,43 @@ class CreateReview(plugin.MayaCreator):
     useMayaTimeline = True
     panZoom = False
 
+    # Overriding "create" method to prefill values from settings.
+    def create(self, subset_name, instance_data, pre_create_data):
+
+        members = list()
+        if pre_create_data.get("use_selection"):
+            members = cmds.ls(selection=True)
+
+        project_name = get_current_project_name()
+        asset_doc = get_asset_by_name(project_name, instance_data["asset"])
+        task_name = get_current_task_name()
+        preset = lib.get_capture_preset(
+            task_name,
+            asset_doc["data"]["tasks"][task_name]["type"],
+            subset_name,
+            get_project_settings(project_name),
+            self.log
+        )
+        self.log.debug(
+            "Using preset: {}".format(
+                json.dumps(preset, indent=4, sort_keys=True)
+            )
+        )
```

Comment:
I think this `preset` data remains unused currently? This will likely need to find its way into the Attribute Defs instead instead of here? Or?

Reply:
Instructed by @iLLiCiTiT to do the attribute manipulation in `create`.

Reply:
Fine by me. But currently it doesn't do that right? `preset` appears to be unused?

Reply:
Should be there now

Reply:
I see the changes made here: https://github.com/BigRoy/OpenPype/pull/13/commits/61af8ae8c0a21058dbb300a204eb319749788942

This seems interesting to say the least. On create you are tweaking what is the `default` value of a Creator attribute definition but that's not the default value the user is seeing since it's after clicking create. 

At the point in time at creation when creating is being done does it really still matter whether you're updating the default? Since basically the user already clicked create with actual different values visible in the UI. Wouldn't it already have been converted to the value the user actually entered in the UI? How does this work? And should this even work, this seems dangerous or hacky?

This doesn't seem the right way to do this. @iLLiCiTiT what's your idea on this?

Reply:
We dont have any of the instance attributes exposed to the user when creating, so these default values are set when creating the instance. The user can then change those values when publishing.

Reply:
![cap1](https://github.com/BigRoy/OpenPype/assets/1860085/29592d05-1756-401a-a599-21ff911f854c)
![cap2](https://github.com/BigRoy/OpenPype/assets/1860085/4eb78199-1dfc-4fc7-b05f-62a4a343e1af)